atlas/AtlasMl/atlasml/routers/competency.py

```
@@ -5,11 +5,12 @@
 from scipy.spatial import ConvexHull
 from scipy.spatial.qhull import QhullError
 
-from atlasml.clients.weaviate import get_weaviate_client
+from atlasml.clients.weaviate import get_weaviate_client, CollectionNames
 from atlasml.dependencies import TokenValidator
```

Ruff (0.12.2)

`atlasml.dependencies.TokenValidator` imported but unused

Remove unused import: `atlasml.dependencies.TokenValidator`

(F401)